### ISSUE TITLE
Cannot tab to delete button of the custom icon uploader

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2857]

**Comment by dwalizer** [comment]
PR #2873

**Comment by sudo-may** [comment]
:+1:
